components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -326,8 +326,10 @@ - (void)setItems:(NSArray<UITabBarItem *> *)items {
       itemView.badgeValue = item.badgeValue;
     }
 #if defined(__IPHONE_10_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0)
-    if (item.badgeColor) {
-      itemView.badgeColor = item.badgeColor;
+    if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 10.0) {
```

Comment:
We have to do all 3 of these conditions?

Reply:
#if is for codecov/Travis. UIDevice is for iOS 8, that doesn't seem to accept @available.